ethstorageproof/ethstorageproof_test.go

```diff
@@ -153,8 +154,18 @@ func TestVerify(t *testing.T) {
 	}
 
 	for i, tt := range tests {
+		var err error
+		var value []byte
+		if len(tt.value) != 0 {
+			value, err = rlp.EncodeToBytes(tt.value)
+			if err != nil {
+				panic(err)
```

Comment:
We usually use t.Fatal() instead of panic()

Reply:
Actually we are slowly migrating to quicktest

`import qt "github.com/frankban/quicktest"`

`qt.Assert(b, err, qt.IsNil)`

But if you don't want to change it on this PR we could do it in the future.

Reply:
I've added an issue to migrate to quicktest: https://github.com/vocdoni/storage-proofs-eth-go/issues/5